apps/docs/src/guide/types/date-time.md

```diff
@@ -0,0 +1,60 @@
+# Date Time
+
+To allow for easier manipulation of date and time, the SDK exports the `DateTime` class, which is a wrapper around the `Date` class ([reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)). Below we will go over the methods of instantiation, utility functions, and time formats.
+
+Why is this useful?
+Internally the transactions and other time/date assets are encoded using the **TAI64** format. We return a `IDateTime` interface, to allow of easier conversion and formatting between the two formats.
+
+## API
+
+We have a host of static method for **instantiation** of our `DateTime` class.
+
+<<< @/../../docs-snippets/src/guide/types/date-time.test.ts#create-from-multiple-sources{ts:line-numbers}
+
+### TAI64
+
+`fromTai64` is a _static_ method, that allows the creation of `DateTime` class from a TAI64 string.
+
+`toTai64` is a _instance_ method, that allows the conversion of a `DateTime` class to a `TAI64` string.
+
+<<< @/../../docs-snippets/src/guide/types/date-time.test.ts#from-tai-64-and-to-tai-64{ts:line-numbers}
+
+### UNIX
+
+`fromUnixMilliseconds` is a _static_ method, that allows the creation of `DateTime` class from a UNIX Milliseconds number.
+
+`toUnixMilliseconds` is a _instance_ method, that allows the conversion of a `DateTime` class to a `UNIX` number in milliseconds.
+
+<<< @/../../docs-snippets/src/guide/types/date-time.test.ts#from-unix-milliseconds-and-to-unix-milliseconds{ts:line-numbers}
+
+`fromUnixSeconds` is a _static_ method, that allows the creation of `DateTime` class from a UNIX Seconds number.
+
+`toUnixSeconds` is a _instance_ method, that allows the conversion of a `DateTime` class to a `UNIX` number in seconds.
+
+<<< @/../../docs-snippets/src/guide/types/date-time.test.ts#from-unix-seconds-and-to-unix-seconds{ts:line-numbers}
+
+### Date
+
+The `DateTime` class extends the functionality of the `Date` object (see here), so all method are available for your usages.
+
+<<< @/../../docs-snippets/src/guide/types/date-time.test.ts#date-object-methods{ts:line-numbers}
+
+## Formats
+
+Here we will go over the different date/time formats that we use in the SDK. Internally the blockchain uses the **TAI64** format, but we also support the **UNIX** format for ease of use.
+
+### UNIX Format
+
+UNIX time is the number of seconds that have elapsed since **00:00:00 Coordinated Universal Time (UTC), Thursday, 1 January 1970**, minus leap seconds. Every day is treated as if it contains exactly 86400 seconds, so leap seconds are ignored.
+
+### TAI Format
+
+TAI stands for _Temps Atomique International_ and is the current international real-time standard [Source](https://cr.yp.to/libtai/tai64.html).
+
+We use **TAI64** is a 64-bit integer representing the number of nanoseconds since the epoch.
```

Comment:
```suggestion
`TAI64` is a 64-bit integer representing the number of nanoseconds since the epoch.
```

I would also recommend replacing all `**TAI64**` with `TAI64` (<code> markdown) instead

Reply:
I concur, that is better 😍